0306.scm

```diff
@@ -31,11 +31,36 @@
 
 
 ;Q3. write multirember
+(define multirember
+  (lambda (a lat)
+    (cond
+      ((null? lat) (quote ()))
+        (else
+          (cond
+            ((eq? (car lat) a)
+              (multirember a (cdr lat)))
+            (else (cons (car lat)
+                  (multirember a (cdr lat)))))))))
+(print (multirember 'a '(a b c a)))
+
 
 ;(print (rember 'a '(a b c a)))
 
 
 
 ;Q4. write reber*
+(define rember*
+  (lambda (a l)
+    (cond
+      ((null? l) (quote ()))
+      ((atom? (car l))
+        (cond
+          ((eq? (car l) a)
+            (rember* a (cdr l)))
+            (else (cons (car l)
+              (rember* a(cdr l))))))
+              (else (cons (rember* a(car l))
```

Comment:
does the indent of this line is current?
and also forget space between `a` and `(car l)`